osu.Game.Rulesets.Osu/Skinning/Default/PlaySliderBody.cs

```diff
@@ -35,11 +37,29 @@ private void load(ISkinSource skin, DrawableHitObject drawableObject)
             accentColour = drawableObject.AccentColour.GetBoundCopy();
             accentColour.BindValueChanged(accent => updateAccentColour(skin, accent.NewValue), true);
 
+            SnakingOut.BindTo(snakingOut);
             config?.BindWith(OsuRulesetSetting.SnakingInSliders, SnakingIn);
-            config?.BindWith(OsuRulesetSetting.SnakingOutSliders, SnakingOut);
+            config?.BindWith(OsuRulesetSetting.SnakingOutSliders, snakingOut);
```

Comment:
is this intentionally no longer binding to the bindable of the underlying class? if so it needs explanation (and probably a variable rename of this private instance to better describe why it is different).

Reply:
oh, it is just above.. but also creating a private bindable for unknown reasons.

Reply:
It's creating a private bindable so that the `TrackFollowCircle` property doesn't affect the configuration bindable. So the config bindable is bound privately, and `SnakingOut` is unbound + mutated locally in the `!TrackFollowCircle` case.

Reply:
Right. I guess renaming the private to `configSnakingOut` would help.